### PR TITLE
Closes #2694: `extrema_test.py` conversion for new test framework

### LINKED ISSUE #2694: `extrema_test.py` conversion for new test framework
Covert `extrema_test.py` to new test framework

### PR DESCRIPTION
**Pull request by pierce314159**
This PR (closes #2694) converts `extrema_test.py` to new test framework

### CODE REVIEW

File: PROTO_tests/tests/extrema_test.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import numpy as np
+import pytest
+
+import arkouda as ak
+
+
```
Comment (TheOddCzar789) [inline PROTO_tests/tests/extrema_test.py:6]: isort wants to remove one of these blank lines

**Review by TheOddCzar789 — APPROVED** [review]

**Review by Ethan-DeBandi99 — COMMENTED** [review]
Looks good. Just one question that may not be needed here.

File: PROTO_tests/tests/extrema_test.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+NO_UINT = ["int64", "float64", "bool"]
+
+
+def make_np_arrays(size, dtype):
+    if dtype == "int64":
+        return np.random.randint(-(2**32), 2**32, size=size, dtype=dtype)
+    elif dtype == "float64":
+        return np.random.uniform(-(2**32), 2**32, size=size)
```
[the remark this answers is not in the document]
Comment (pierce314159) [reply PROTO_tests/tests/extrema_test.py:14]: Ran into issue while adding this. It seems this isn't working for the extrema methods or for sorting. Added #2703 to track this

File: PROTO_tests/tests/extrema_test.py (hunk 1 above)
Comment (pierce314159) [reply PROTO_tests/tests/extrema_test.py]: reran isort

**Review by Ethan-DeBandi99 — APPROVED** [review]

**Pull request merged**